confignode/src/main/java/org/apache/iotdb/confignode/consensus/request/write/DeleteStorageGroupReq.java

```diff
@@ -18,40 +18,40 @@
  */
 package org.apache.iotdb.confignode.consensus.request.write;
 
-import org.apache.iotdb.commons.utils.BasicStructureSerDeUtil;
+import org.apache.iotdb.commons.utils.ThriftConfigNodeSerDeUtils;
 import org.apache.iotdb.confignode.consensus.request.ConfigRequest;
 import org.apache.iotdb.confignode.consensus.request.ConfigRequestType;
+import org.apache.iotdb.confignode.rpc.thrift.TStorageGroupSchema;
 
 import java.nio.ByteBuffer;
 import java.util.Objects;
 
 public class DeleteStorageGroupReq extends ConfigRequest {
 
-  private String storageGroup;
+  private TStorageGroupSchema storageGroup;
```

Comment:
Why change this parameter? Isn't it enough to pass the StorageGroupName for the deletion request?

Reply:
Because we need to delete region entries in regionMap by  region ids, and  delete partition entries in partition map by storageGroupName. It is more convienent to use this parameter. And here is the same object in DeleteStorageGroupProcedure.

![image](https://user-images.githubusercontent.com/82880298/167242277-07509da2-d622-4efd-b017-a9ae9e9847c7.png)

Reply:
But the DeleteStorageGroupsReq uses List<String> storageGroups, why?